app/(main)/agents/page.tsx

```diff
@@ -1,55 +1,66 @@
-import { playgroundV2Flag } from "@/flags";
-import { fetchCurrentUser } from "@/services/accounts";
-import { createAgent, getAgents } from "@/services/agents";
-import { CreateAgentButton } from "@/services/agents/components";
+import { agents, db } from "@/drizzle";
 import { fetchCurrentTeam } from "@/services/teams";
-import { redirect } from "next/navigation";
-import { Suspense } from "react";
-
-type AgentListProps = {
-	teamDbId: number;
-};
-async function AgentList(props: AgentListProps) {
-	const agents = await getAgents({ teamDbId: props.teamDbId });
+import { and, eq, isNotNull } from "drizzle-orm";
+import Link from "next/link";
+import { type ReactNode, Suspense } from "react";
+import { formatTimestamp } from "../../(playground)/p/[agentId]/lib/utils";
 
+function DataList({ label, children }: { label: string; children: ReactNode }) {
 	return (
-		<div className="flex flex-col gap-2">
-			{agents.map(({ id, name }) => (
-				<a key={id} className="flex border border-border p-4" href={`/p/${id}`}>
-					{name ?? id}
-				</a>
-			))}
+		<div className=" text-black-30">
+			<p className="text-[12px]">{label}</p>
+			<div className="font-bold">{children}</div>
 		</div>
 	);
 }
-export default async function AgentListPage() {
-	const enableV2 = await playgroundV2Flag();
-	if (enableV2) {
-		return redirect("/agents-v2");
-	}
-	const currentUser = await fetchCurrentUser();
+
+async function AgentList() {
 	const currentTeam = await fetchCurrentTeam();
-	async function createAgentAction() {
-		"use server";
-		const agent = await createAgent({
-			teamDbId: currentTeam.dbId,
-			creatorDbId: currentUser.dbId,
-		});
-		redirect(`/p/${agent.id}`);
+	const dbAgents = await db
+		.select({ id: agents.id, name: agents.name, updatedAt: agents.updatedAt })
+		.from(agents)
+		.where(
+			and(eq(agents.teamDbId, currentTeam.dbId), isNotNull(agents.graphUrl)),
+		);
+	if (dbAgents.length === 0) {
+		return (
+			<div className="flex justify-center items-center h-full">
+				<div className="grid gap-[12px] justify-center text-center">
+					<div>No agents found</div>
+				</div>
+			</div>
+		);
 	}
 	return (
-		<div className="container mt-8">
-			<section className="text-foreground">
-				<div className="flex flex-col gap-8">
-					<div className="flex justify-between">
-						<h1>Agents</h1>
-						<CreateAgentButton createAgentAction={createAgentAction} />
+		<div className="grid gap-[16px] grid-cols-3">
+			{dbAgents.map((agent) => (
+				<Link href={`/p/${agent.id}/canary`} key={agent.id}>
```

Comment:
```suggestion
				<Link href={`/p/${agent.id}`} key={agent.id}>
```

Reply:
@toyamarinyon I cannot open the agent page in `/agents`. Please check it and try your OAT.